lib/previously_changed.rb

```diff
@@ -4,6 +4,6 @@ module PreviouslyChanged
   extend ActiveSupport::Concern
 
   def previously_changed?(attribute)
-    saved_change_to_attribute?(attribute)
+    Array(previous_changes[attribute.to_s]).any?
```

Comment:
This was
```suggestion
    saved_change_to_attribute?(attribute.to_s])
```

Reply:
https://github.com/3scale/porta/commit/8f8e86e65e9e7756d9313b01a29ea7ebf6b26ae4#diff-8b57a68842938408d106ab5411d0190f4cc46684908d0cdb9a11503e1971c4eb
```diff
-saved_change_to_attribute?(attribute.to_s])
+saved_change_to_attribute?(attribute)
```

Reply:
it is from 1256260ba94d34d6a7a44fdb233a3fdd3a21ea55